hathor/utils/txt.py

```diff
@@ -0,0 +1,93 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import ast
+import os
+from pathlib import Path
+from typing import Any, Union
+
+
+def dict_from_file(*, filepath: Union[Path, str]) -> dict[str, Any]:
```

Comment:
Why don't we split the parser from `SysctlProtocol.lineReceived()` to a separate `hathor/sysctl/parser.py` and use it?